jupyterlab_bigquery/.eslintrc.js

```diff
@@ -0,0 +1,28 @@
+/**
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+const esLintConfig = require('../.eslintrc');
+
+esLintConfig.rules = {
```

Comment:
This is fine to include for now, but where possible we should use the same linting configuration and coding style across all of the subpackages.

Reply:
What linting config/coding style are the other subpackages following currently? Is there a unified way or has it yet to be determined?

Reply:
Basically the rules defined at the repo root should be pretty standard and are in compliance with Google TS style rules for the most part. I think the GCS extension had to deviate from these in a few places because it was forked from the base filebrowser extension.